src/RemoteTech/FlightComputer/Commands/ActionGroupCommand.cs

```diff
@@ -17,9 +17,9 @@ public override bool Pop(FlightComputer f)
             f.Vessel.ActionGroups.ToggleGroup(ActionGroup);
             if (ActionGroup == KSPActionGroup.Stage && !(f.Vessel == FlightGlobals.ActiveVessel && FlightInputHandler.fetch.stageLock))
             {
-                Staging.ActivateNextStage();
-                ResourceDisplay.Instance.Refresh();
-            }
+				KSP.UI.Screens.StageManager.ActivateNextStage();
+				KSP.UI.Screens.ResourceDisplay.Instance.Refresh();
+			}
```

Comment:
I know we've talked about it on Slack, but just to document it: indentation seems to be off in some parts of this PR. This is one example.